app/src/common/shared/com/igalia/wolvic/ui/widgets/Windows.java

```diff
@@ -1470,6 +1473,38 @@ public void openInKioskMode(@NonNull String aUri) {
         mFocusedWindow.setKioskMode(true);
     }
 
+    public void openInImmersiveMode(Uri targetUri, String immersiveParentElementXPath, String immersiveTargetElementXPath) {
+        String extensionId = "wolvic-autowebxr@igalia.com";
+        String extensionUrl = "resource://android/assets/extensions/wolvic_autowebxr/";
```

Comment:
Can we move these two to the top of the file with the other declarations

Reply:
Yes, good point, and they should be constants.